types/environment/container.go

```diff
@@ -18,13 +18,20 @@ package environment
 
 // Container stores container based execution information
 type Container struct {
-	Image          string         `yaml:"image"`
-	WorkingDir     string         `yaml:"workingDir,omitempty"`
-	ContainerBuild ContainerBuild `yaml:"build"`
+	// ID is the id of the running container.
+	ID string `yaml:"-"`
```

Comment:
```
The Container class is not referencing a running instance of a container, so it might be better to keep the ID out of container struct.
```
I know but it is referencing a running instance now.